reana_server/reana_admin/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2022 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""REANA-Server admin tools."""
+
+from .main import reana_admin
```

Comment:
Maybe we can call the file `cli.py` instead of `main.py`, like we do for [`reana-dev`](https://github.com/reanahub/reana/blob/7bddf3f23bb7afba3a363e0413d1ca6414e2b214/reana/reana_dev/cli.py)?

Reply:
Really good suggestion. I will change to `cli.py`